src/exe_rules.ml

```diff
@@ -86,6 +96,13 @@ let executables_rules ~sctx ~dir ~dir_kind ~expander
   let cctx =
     let requires_compile = Lib.Compile.direct_requires compile_info in
     let requires_link = Lib.Compile.requires_link compile_info in
+    let js_of_ocaml =
+      let js_of_ocaml = exes.buildable.js_of_ocaml in
+      if Dune_project.explicit_js_mode (Scope.project scope) then
```

Comment:
We already obtain the value when computing the linkages so let's just reuse that.

Reply:
Fixed, thanks.